services/ui-src/src/components/menus/Menu.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from "@testing-library/react";
+import { RouterWrappedComponent } from "utils/testing/setupJest";
+import { Menu } from "components";
+import { testA11y } from "utils/testing/commonTests";
+
+const menuComponent = (
+  <RouterWrappedComponent>
+    <Menu handleLogout={() => {}} />
+  </RouterWrappedComponent>
+);
+
+describe("<Menu />", () => {
+  test("Menu button is visible", () => {
+    render(menuComponent);
+    expect(screen.getByTestId("header-menu-dropdown-button")).toBeVisible();
```

Comment:
I _think_ you'll be able to find this element by `aria-label` instead of `data-testid`. Test ID should generally be a last resort for locating elements in tests, because we're trying to emulate how users interact with the page. They can see (or hear) text and labels, but they'll never check the DOM for test IDs.

Reply:
I do recommend reading through the "Which Query Should I Use" page of the Testing Library docs: https://testing-library.com/docs/queries/about#priority

Reply:
Oooh good tidbit. I've noticed we frequently use `getbyTestId` throughout MFP. Now's a good time to address this across our initial tests while HCBS is still fresh :D